draft-keytrans-mcmillion-protocol.md

```diff
@@ -514,22 +513,20 @@ struct {
 } PrefixParent;
 ~~~
 
-where `Hash.Nh` is the output length of the ciphersuite hash function. The value
-of a parent node is computed by hashing together the values of its left and
-right children:
+where `Hash.Nh` is the output length of the ciphersuite hash function.
 
-~~~ pseudocode
-parent.value = Hash(0x01 ||
-                   nodeValue(parent.leftChild) ||
-                   nodeValue(parent.rightChild))
```

Comment:
The reason to explicitly say `parent.value = ` is that `value` is an actual field of `PrefixParent` that needs to be populated, potentially stored on-disk. That connection is kind of lost when you roll it into `nodeValue`. And unlike `PrefixLeaf`, `value` can't be computed in isolation

Reply:
Same as above.